boilerplate/_lib/common.sh

```diff
@@ -183,6 +183,8 @@ if [ -z "$BOILERPLATE_GIT_REPO" ]; then
   export BOILERPLATE_GIT_REPO=https://github.com/openshift/boilerplate.git
 fi
 
+# Base image repo url
+IMAGE_REPO=quay.io/redhat-services-prod/openshift
 # The namespace of the ImageStream by which prow will import the image.
 IMAGE_NAMESPACE=openshift
```

Comment:
Is this `IMAGE_NAMESPACE` variable needed as I can see it's not used anywhere in the script? or should we use it to replace IMAGE_REPO as `IMAGE_REPO=quay.io/redhat-services-prod/$IMAGE_NAMESPACE`

Reply:
This PR is not touching that variable. This file is sourced from other scripts where it is used.